packages/webapp/next.config.js

```diff
@@ -91,6 +91,11 @@ module.exports = withTM(
               source: '/posts/finder',
               destination: '/search?provider=posts',
               permanent: false
+            },
+            {
+              source: '/signup',
+              destination: '/onboarding',
+              permanent: true
```

Comment:
Do we want it permanent `/signup` is a common path so in the future we might want to do something else on it?

Reply:
Don't think it's much of a problem. It would only affect new users, and if we start using signup page again, new users (never visited before), won't have ever visited the redirect before. 

But I can change it to temporary.

Reply:
I'll just change it to temporary.

Reply:
Done, it has been updated in #2630